README.md

```diff
@@ -74,65 +69,63 @@ Windows Terminal is a new, modern, feature-rich, productive terminal application
 
 The Terminal will also need to meet our goals and measures to ensure it remains fast and efficient, and doesn't consume vast amounts of memory or power.
 
-### The Windows Console host
-
-The Windows Console host, `conhost.exe`, is Windows' original command-line user experience.
-
-The Console is the app you see and interact with when you run Cmd, PowerShell, WSL, or any other command-line application on Windows.
-
-Console also implements Windows' command-line infrastructure, hosting the Windows Console API, input engine, rendering engine, and user preferences. 
+### The Windows Console Host
 
-> The Console's code in this repository is the actual source from which the `conhost.exe` in Windows itself is built.
+The Windows Console host, `conhost.exe`, is Windows' original command-line user experience. It also hosts Windows' command-line infrastructure and the Windows Console API server, input engine, rendering engine, user preferences, etc. The console host code in this repository is the actual source from which the `conhost.exe` in Windows itself is built.
 
-Since taking ownership of the Windows command-line in 2014, the team has added several new features to the Console, including window transparency, line-based selection, support for [ANSI / Virtual Terminal sequences](https://en.wikipedia.org/wiki/ANSI_escape_code), [24-bit color](https://devblogs.microsoft.com/commandline/24-bit-color-in-the-windows-console/), a [Pseudoconsole ("ConPTY")](https://devblogs.microsoft.com/commandline/windows-command-line-introducing-the-windows-pseudo-console-conpty/), and more.
+Since taking ownership of the Windows command-line in 2014, the team added several new features to the Console, including background transparency, line-based selection, support for [ANSI / Virtual Terminal sequences](https://en.wikipedia.org/wiki/ANSI_escape_code), [24-bit color](https://devblogs.microsoft.com/commandline/24-bit-color-in-the-windows-console/), a [Pseudoconsole ("ConPTY")](https://devblogs.microsoft.com/commandline/windows-command-line-introducing-the-windows-pseudo-console-conpty/), and more.
 
-However, because Windows Console's primary goal is to maintain backward compatibility, we've been unable to add many of the features the community has been asking for (and which we've been wanting to add) for the last several years including tabs, unicode text, and emoji.
+However, because Windows Console's primary goal is to maintain backward compatibility, we have been unable to add many of the features the community (and the team) have been asking for for the last several years including tabs, unicode text, emoji, etc.
```

Comment:
I recall bringing up the wording complication in having both _including_ and _etc_. You _have to choose one._ Why did this come back?

Reply:
Sorry - my bad. Lost in a merge conflict. Fixed.